app/index.js

```diff
@@ -203,9 +204,10 @@ async function playNotificationSound(_event, options) {
 	console.debug('No notification sound played', player, options);
 }
 
-function onRenderProcessGone(event, details) {
-	console.error(`render-process-gone with reason: '${details.reason}'.`);
-	console.error(`Event '${event}' and details '${details}'`);
+function onRenderProcessGone(event, webContents, details) {
+	// https://www.electronjs.org/docs/latest/api/app#event-render-process-gone
```

Comment:
I don't think we need this commented lines. Is ok to remove them

Reply:
Sure